vb-fe/src/util/Room.ts

```diff
@@ -6,10 +6,30 @@ export interface RoomForm {
   tags: Tag[];
 }
 
+export interface dataCreateRoom {
+  description: string;
+  host: {
```

Comment:
make this `host: User` (user being another interface)

Reply:
like it is with `Tag`

Reply:
also what is this used for?

Reply:
this is just what the backend returns as the room object from the create room endpoint, since we `populate` everything

Reply:
So call this `Room` (what the backend returns). and call the other one CreateRoom/RoomForm and take only the required things from this interface. Try to keep interface names generic because the whole point of interfaces is reusability and giving multiple variables a structure